### PR TITLE
Add changes to com.ibm.ws.org.apache.cxf.ws.security.2.6.2 into Open-Liberty

### PR DESCRIPTION
**Pull request by BerksanAtes**
for #9765
Bundle wired to use CXF trough internal bundles. Plus open-liberty specific changes

### CODE REVIEW

**Comment by BerksanAtes** [comment]
#build

File: dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.overrides [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+cxfVersion=2.6.2
+
+Bundle-Name: com.ibm.ws.org.apache.cxf.ws.security.2.6.2
+Bundle-Description: cxf ws security, version 2.6.2
+Bundle-SymbolicName: com.ibm.ws.org.apache.cxf.ws.security.2.6.2
+Bundle-ActivationPolicy: lazy
+
+
+Export-Package: \
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.overrides:24]: This is lacking a version number. You should look at the original version of the org.apache.cxf.ws.security.2.6.2/META-INF/Manifest.mf file and make sure that the packages listed under it's Export-Package header match what's being exported in the Export-Package header of this bnd.overrides file. Do not use a org.apache.cxf.ws.security.* wild card, rather explicitly list each individual package with it's version number also listed.

File: dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.overrides [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  org.apache.cxf.ws.security.*
+
+Import-Package:  \
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.overrides:27]: Similar to the Export-Package, make sure the import statement matches exactly. I don't see a `*` import so be very careful to make sure you're importing every package that's needed by this bundle. If not, runtime behavior may be effected.

**Comment by BerksanAtes** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.wssecurity_fat, com.ibm.ws.wssecurity_fat.cxf, com.ibm.ws.wssecurity_fat.saml, com.ibm.ws.wssecurity_fat.saml.2, com.ibm.ws.wssecurity_fat.twas

**Comment by andrea-pichler** [comment]
No L2 message review needed for translations

**Comment by BerksanAtes** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.wssecurity_fat.cxf, com.ibm.ws.wssecurity_fat.saml, com.ibm.ws.wssecurity_fat.saml.2, com.ibm.ws.wssecurity_fat.twas

**Comment by BerksanAtes** [comment]
#libbybot

**Pull request merged**